src/screens/FeedScreen/RenderList.js

```diff
@@ -296,7 +309,9 @@ RenderListFeed.propTypes = {
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
   onHeaderOptionClicked: PropTypes.func,
-  onPreviewCommentPress: PropTypes.func
+  onPreviewCommentPress: PropTypes.func,
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New PropTypes `onDeletePost` and `isSelf` have been added. This is good for type checking but make sure that these props are always provided where this component is used to avoid any runtime errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->